src/packager.ts

```diff
@@ -172,7 +165,7 @@ export class Packager implements BuildInfo {
     }
     else {
       log("Skipping app dependencies installation because dev and app dependencies are not separated")
-      return Promise.resolve(null)
+      return BluebirdPromise.resolve(null)
```

Comment:
[Performance and memory consumption](http://programmers.stackexchange.com/a/279003) is not a problem, but lack of [long stack traces](http://bluebirdjs.com/docs/features.html#long-stack-traces) stop us to use native promises.

Also, bluebird provides [coroutine](http://bluebirdjs.com/docs/api/promise.coroutine.html) and a lot of useful methods (promisify, map).